tests/provision_test.go

```diff
@@ -57,7 +57,14 @@ func blockVolCreationTest() {
 	By("Deleting storage class", deleteStorageClass)
 }
 
+func volCreationFailTest() {
+	By("Create and Verify DeviceVol resource", CreateDeviceVolume)
```

Comment:
q:  We usually dont create the DeviceVolume resource, it is created by the controller right? 

So should we add a test case like that, starting by creating a PVC which can result in a failed DeviceVolume. Is it possible to create such a test case?

Reply:
@akhilerm I did try that approach as well, But by the time we try to get the DeviceVolume resource, it gets deleted by the controller.